services/oauth/src/main/java/org/mule/services/oauth/internal/builder/DefaultOAuthAuthorizationCodeDancerBuilder.java

```diff
@@ -140,12 +140,12 @@ public void dispose() {
 
       @Override
       public RequestHandlerManager addRequestHandler(String path, RequestHandler requestHandler) {
-        return server.addRequestHandler(localCallbackConfigPath, requestHandler);
+        return server.addRequestHandler(path, requestHandler);
```

Comment:
Can the code that was doing the no lifecycle and the counting server stuff be removed?